src/components/footer.jsx

```diff
@@ -9,17 +9,22 @@ import { useTranslate } from 'react-translate'
 function Footer () {
   const t = useTranslate('Conticki')
   const context = useContext(AppStateContext)
+  const conference = context.conference.get
 
   return (
     <Container fluid className='bg-secondary'>
       <Row>
         <Col>
           <Container>
             <Row className='my-3'>
-              <Col md={6} className=''>
+              <Col md={4}>
                 <p className='text-light mt-2'>{t('OS2Contiki - Conference and ticket management')}</p>
               </Col>
-              <Col md={6}>
+              <Col md={4}>
+                {conference.app.accessibility_statement?.url &&
+                  <p className='mt-2'><a className='text-light' href={conference.app.accessibility_statement.url}>{conference.app.accessibility_statement.text ? conference.app.accessibility_statement.text : t('Accessibility statement')}</a></p>}
```

Comment:
Using `{conference.app.accessibility_statement.text || t('Accessibility statement')}` will probably make this easier to read (and we do it like in all other places).